JSDemos/Demos/ButtonGroup/Overview/description.md

```diff
@@ -1,3 +1,17 @@
-ButtonGroup is a UI component that displays a set of toggleable buttons and allows a user to select one or several of them. This demo illustrates how to configure the ButtonGroup to enable single and multiple selection.
+ButtonGroup displays a set of toggleable buttons and allows users to select one or several of them. This demo illustrates how to configure the ButtonGroup: bind ButtonGroup to data, enable or disable multiple selection and customize the ButtonGroup appearance.
```

Comment:
> ButtonGroup displays a set of toggleable buttons and allows users to select one or **several** of them.

If memory serves, I already told you last time that we don't use the word "several" because it's more of a colloquial term. We say "multiple" instead.

Second. "Toggleable" is probably not the best choice of a term...

```suggestion
ButtonGroup displays a set of two-state buttons and allows users to select one or multiple buttons. This demo illustrates how to configure the ButtonGroup in the following steps:

* Bind ButtonGroup to data
* Enable or disable multiple selection
* Customize the ButtonGroup appearance
```